charts/sscs-tribunals-api/values.ccd.preview.template.yaml

```diff
@@ -374,7 +374,7 @@ sscs-evidence-share:
       CASE_ACCESS_MANAGEMENT_FEATURE: true
 
 sscs-ccd-callback-orchestrator:
-  enabled: false
+  enabled: true
```

Comment:
Set this back to false, by default for preview envs we don't want to spin up containers unless the dev deploying needs it.